packages/react/src/components/UIShell/components/styles/_side-nav.scss

```diff
@@ -568,13 +586,7 @@ div:has(.#{$prefix}--header)
 
 // allow overlay to display at all breakpoints
 .#{$prefix}--side-nav__overlay-active {
-  z-index: z('overlay');
-  background-color: $overlay;
-  block-size: 100vh;
-  inline-size: 100vw;
-  opacity: 1;
-  transition: opacity $transition-expansion $standard-easing,
-    background-color $transition-expansion $standard-easing;
+  background-color: transparent;
```

Comment:
fyi: if i use the prop `hideOverlay` then the `SideNav` will no longer close when a user clicks on the main content